composer.json

```diff
@@ -31,35 +31,35 @@
     "prefer-stable": true,
     "minimum-stability": "dev",
     "require": {
-        "php": ">=7.0"
+        "php": ">=7.2"
```

Comment:
Should we really bump the min-php version? Did you encountered any issue with it for this PR?

Reply:
i got a problem with the return type in phpunit classes

Reply:
> 1) Elao\Enum\Tests\Integration\Bridge\Doctrine\DBAL\Type\EnumTypeTest::testEnumType
TypeError: Return value of Elao\Enum\Tests\Integration\Bridge\Doctrine\DBAL\Type\EnumTypeTest::setUp() must be an instance of Elao\Enum\Tests\Integration\Bridge\Doctrine\DBAL\Type\void, none returned